index.d.ts

```diff
@@ -18,9 +18,13 @@ export type Options = {
 	readonly basePath?: string;
 
 	/**
-	Remove any paths that match this regexp from the stack.
+	Remove any paths that return false from this callback.
+
+  Example with `path => /unicorn/.test(path)` as `pathFilterCallback`:
+
+	`/Users/sindresorhus/dev/clean-stack/unicorn.js:2:15` → ``
```

Comment:
I think a full example would be useful in this case. Something like:


@example
```
import cleanStack from 'clean-stack';

const error = new Error('Missing unicorn');

console.log(cleanStack(error.stack));
// Error: Missing unicorn
//     at Object.<anonymous> (/Users/sindresorhus/dev/clean-stack/unicorn.js:2:15)
//     at Object.<anonymous> (/Users/sindresorhus/dev/clean-stack/unicorn.js:2:15)

console.log(cleanStack(error.stack, {pathFilter ...}));
// Error: Missing unicorn
//     at Object.<anonymous> (/Users/sindresorhus/dev/clean-stack/unicorn.js:2:15)
```